data/endpoints/accountsEndpoints.js

```diff
@@ -0,0 +1,92 @@
+//import express from the express dependency
+const express = require('express');
+
+//setup express router from express
+const router = express.Router();
+
+//import database from the seeds folder?
+const db = require('../dbConfig');
+
+
+//<----------GET REQUESTS-------------------
+router.get('/', (req, res) => {
+
+    //selects ALL from the database
+    db.select('*')
+        .from('accounts')
+        .then(accounts => {
+            res.status(200).json(accounts);
+        })
+        .catch(err => {
+            res.json(err);
+        })
+    })
+    
+    router.get('/:id', (req, res) => {
+        
+        //selects the id's only 
+        db.select('id')
+            .from('accounts')
+            .then(id => {
```

Comment:
you want to add a .first() in here so if there ever ends up being multiple values it only takes the very first one